.azurePipeline/getPodsLogs.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+The script will currently be used in the Azure Pipeline to get back the logs of the pods when a test is failing
+allowing developers to document the issue.
+"""
+
+import json
+from pathlib import Path
+import subprocess
+import sys
+
+# The namespace is not configurable as we are expected to run this script only in Azure Dev-Ops which deployed the
+# service only in this namespace.
+NAMESPACE = "test-azure"
+
+
+def get_list_pods(_release_name):
+    cmd = "kubectl --namespace {} get pods -l release={} -o json".format(NAMESPACE, _release_name)
+    r = subprocess.run(cmd.split(" "), capture_output=True)
+    items_from_release = json.loads(r.stdout)['items']
+    return items_from_release
+
+
+def get_logs_container(container_info, file, previous=False):
+    cmd = "kubectl --namespace {} logs {} {}".format(NAMESPACE, container_info['full_pod_name'], container_info['name'])
+    if previous:
+        cmd += " --previous"
+    subprocess.call(cmd.split(" "), stdout=file)
+
+
+def info_from_pod(_pod, _directory):
+    pod_name = _pod.get('metadata').get('name')
+    list_containers = _pod.get('status').get('containerStatuses')
+    for container in list_containers:
+        name_container = container.get('name')
+        restart_count = container.get('restartCount')
+        image_name = container.get('image')
+        info = {'full_pod_name': pod_name, 'short_pod_name': pod_name[len('benchmark-es-data61-xyz-'):],
```

Comment:
`benchmark-es-data61-xyz-`?

Reply:
That explains why the short pod name was quite small... :+1: